requirements.txt

```diff
@@ -5,7 +5,7 @@ pexpect==4.8.0
 sentence-transformers==2.2.2
 #torch==1.12.1
 #transformers==4.21.2
-sexpdata
+sexpdata==1.0.0
```

Comment:
sexpdata’s latest version seems to have some compatibility issues, so I had to downgrade it